server/src/main/java/org/kaazing/gateway/server/config/parse/translate/AddServiceNameVisitor.java

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
I'm specifically talking about the AddServiceNameVisitor class being moved out of CE.